cc.arduino.IDE2.json

```diff
@@ -0,0 +1,117 @@
+{
+    "app-id": "cc.arduino.IDE2",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "arduino-ide",
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--share=network",
+        "--device=all",
+        "--filesystem=home",
+        "--filesystem=xdg-run/keyring",
+        "--system-talk-name=org.freedesktop.DBus",
```

Comment:
I'm not sure this is the right value either.

Reply:
from what i can see the application talks with various dbus functions and adding each one individually seems like a lot of unneeded effort. i had a look through some of the other apps on flathub and they dont appear to do that.

Reply:
some examples include:
`C17: -> org.freedesktop.DBus call org.freedesktop.DBus.ListNames at /org/freedesktop/DBus
C18: -> org.freedesktop.DBus call org.freedesktop.DBus.RemoveMatch at /org/freedesktop/DBus
C16: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
`

Reply:
ok so after monitoring my dbus i believe i've figured out what dbus names i need to add so i am going to resolve this and i have pushed a commit with the new dbus name.